source/help/getting-started/signing-in.rst

```diff
@@ -97,14 +90,14 @@ Mattermost.
 Switching Teams
 ---------------
 
-You can switch between teams you have joined using the team sidebar
-that appears left of your channel list on the left-hand sidebar.
+You can switch between teams you have joined using the team sidebar that appears to the left of your channel list on the left-hand sidebar.
 
 .. image:: ../../images/team-sidebar.png
```

Comment:
@cwarnermm separate from this PR these images should be updated

Reply:
Thanks, @esethna. I've logged a docs ticket (https://mattermost.atlassian.net/browse/MM-33036) for this update.